go.sum

```diff
@@ -274,6 +272,8 @@ github.com/envoyproxy/go-control-plane v0.9.9-0.20210512163311-63b5d3c536b0/go.m
 github.com/envoyproxy/go-control-plane v0.9.10-0.20210907150352-cf90f659a021/go.mod h1:AFq3mo9L8Lqqiid3OhADV3RfLJnjiw63cSpi+fDTRC0=
 github.com/envoyproxy/protoc-gen-validate v0.1.0/go.mod h1:iSmxcyjqTsJpI2R4NaDN7+kN2VEUnK/pcBlmesArF7c=
 github.com/evanphx/json-patch v4.9.0+incompatible/go.mod h1:50XU6AFN0ol/bzJsmQLiYLvXMP4fmwYFNcr97nuDLSk=
+github.com/fatelei/nydus-snapshotter v0.3.0-alpha.7 h1:3FVE44etcBcDnPcDlaBLDSGZEhmU4XJKQO+pQs6xObw=
```

Comment:
Maybe we can submit a PR to `containerd/nydus-snapshotter` first. :)

Reply:
Someday we should remove the `containerd/nydus-snapshotter` dep.

Reply:
`containerd/nydus-snapshotter` has remove `PrepareMetaLayer` method, so we can using `main` branch